Content/posts/revealing-and-explaining-package-scores.md

```diff
@@ -0,0 +1,63 @@
+
+<picture class="shadow">
+  <source srcset="/images/package-maintainers-score-section~dark.png" media="(prefers-color-scheme: dark)">
+  <img src="/images/package-maintainers-score-section~light.png" alt="The package score section on the package maintainer's page showing a breakdown of package score for a package.">
+</picture>
+
+The most important thing about this package score is that we want it to
+
+While the algorithm for calculating package scores [has always been publicly available](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/blob/main/Sources/App/Core/Score.swift), it’s unreasonable to expect people to browse the source code to see how we score packages. Today’s release makes the scoring process much more transparent.
+
+### Score traits
+
+The package score is currently evaluated based on ten score traits. Previously, we relied on eight attributes to determine a package’s score.
+
+- Whether the package is archived or not (max +20 points)
+- Whether the package has an open-source license that is compatible with the App Store (Up to +10 points)
+- The number of releases (Up to +20 points)
+- The number of stars (Up to +37 points)
+- The number of dependencies (Up to +5 points)
+- The latest activity (Up to +15 points)
+- Whether the package has documentation (Up to +15 points)
+- The number of contributors (Up to +10 points)
+
+### Adding new score traits
+
+Along with introducing the package score section to the maintainer’s page, we have included two additional score traits in our assessment.
+
+- Whether the package has any test targets (Up to 5 points)
+- Whether the package has a README file (Up to 15 points)
+
+There is always room for further improvement to the package score, and these ten traits are just the start of making this score more comprehensive.
+
+### Finding your package’s score
+
+As a package author, if you’d like to see the breakdown of your package score, find the “Do you maintain this package?” section at the bottom of the right-hand sidebar on your package page, and you’ll find the score breakdown at the end.
+
+<picture class=“shadow“>
+  <source srcset=“/images/find-package-maintainers-page~dark.png“ media=“(prefers-color-scheme: dark)“>
+  <img src=“/images/find-package-maintainers-page~light.png“ alt=“The location of the Learn More link that takes package authors to the package maintainer’s page for their package.“>
+</picture>
+
+If you have any feedback regarding the package score or suggestions on how we can improve it, we would love to hear your thoughts in our [discussion forum](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/discussions/2591). We appreciate any contributions as we continue to enhance the package score system.
```

Comment:
I also wanted to add this note to the bottom, but let me know if you feel this is better to include in the Swift Forums blog post instead.

> As a final note to this post, I can't express how thankful I am to have had Dave as my mentor in the Swift Mentorship program. Contributing to this project under his guidance has been an incredible learning experience and I'm grateful for the knowledge, support, and inspiration he has provided me. Thank you Dave!

Reply:
Oops, did I accidentally remove that? 🫨 Sorry. Thank you for the kind words and please feel free to add it back.

Reply:
no problem! updated :)